prac_07/guitars.py

```diff
@@ -0,0 +1,44 @@
+from prac_07.guitar import Guitar
+
+
+def main():
+    # Create guitars list for storing
+    guitars = []
+
+    print("My Guitars!")
+    print("Enter blank name to show guitar list and exit!")
+    name = str(input("Name: "))
+    # Loop as long as user does not input empty name
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_info = Guitar(name, year, cost)
+        print("{self.name} ({self.year}) : ${self.cost:.2f} added.".format(self=guitar_info))
+        guitars.append(guitar_info)
+        # Restart Loop
+        name = input("Name: ")
+
+    # Exit Loop
+    print("... snip ...")
```

Comment:
What's this? In my sample output, it's to show I was cutting the output to save space.